omnibus/config/templates/datadog-agentless-scanner/sysvinit_debian.agentless-scanner.erb

```diff
@@ -0,0 +1,163 @@
+#!/bin/sh
+
+### BEGIN INIT INFO
+# Provides: datadog-agentless-scanner
+# Short-Description: Start and stop datadog agentless scanner
+# Description: Datadog Agentless Scanner
+# Required-Start: $remote_fs
+# Required-Stop: $remote_fs
+# Default-Start: 2 3 4 5
+# Default-Stop: 0 1 6
+### END INIT INFO
+
+. /lib/lsb/init-functions
+
+ETC_DIR="<%= etc_dir %>"
+INSTALL_DIR="<%= install_dir %>"
+AGENTPATH="$INSTALL_DIR/bin/agentless-scanner"
+PIDFILE="$INSTALL_DIR/run/agentless-scanner.pid"
+AGENT_ARGS="--config=$ETC_DIR/datadog.yaml --pidfile=$PIDFILE"
+AGENT_USER="root"
+NAME="datadog-agentless-scanner"
+DESC="Datadog Agentless Scanner"
+
+
+
+if [ ! -x $AGENTPATH ]; then
```

Comment:
could you enclose the variable in double quotes to ensure it doesn't break if the install path contains spaces?